### ISSUE TITLE
every few seconds vlc freeze

### ISSUE THREAD

**Issue by ra1nb0w** [BtbN/vlc-htsp-plugin#18]
Hi,
I am using Gentoo and Vlc compiled from portage (version 2.2.0) and I built htsp from source and then copied
to /usr/lib64/vlc/plugins/access/. the plugin appears on vlc; I receive the channels list from tvheadend 3.9.2706.
When I click the stream starts but it freezes every 2-3 seconds.
I mean, 2-3 seconds of video+audio and then ~1/2 seconds of silence ....
I tried compiling vlc from git but nothing changes; maybe libav or something related to the decoding part?
Do you have some ideas?
thanks

**Comment by BtbN** [comment]
Yes, some change in tvheadend has started this issue. I have no idea what exactly causes it, and even less of an idea how to fix it.
I don't have enough time to analyze this. especialy as i'm no longer actively using this myself.

**Comment by ra1nb0w** [comment]
Ok. Do you remember which release has started this issue?
